taskcat/s3_stage.py

```diff
@@ -0,0 +1,149 @@
+
+class S3APIResponse:
+    def __init__(self, x):
+        self._http_code = x['ResponseMetadata']['HTTPStatusCode']
+
+    @property
+    def ok(self):
+        if self._http_code == 200:
+            return True
+        return False
+
+class S3BucketCreatorException(Exception):
```

Comment:
subclass TaskCat exception to get cli exception handling to realize that this is a handled exception not 500/unhandled

Reply:
:+1: